backend/data/src/main/kotlin/io/tolgee/constants/Message.kt

```diff
@@ -235,6 +235,11 @@ enum class Message {
   SLACK_WORKSPACE_ALREADY_CONNECTED,
   SLACK_CONNECTION_ERROR,
   EMAIL_VERIFICATION_CODE_NOT_VALID,
+  THIRD_PARTY_AUTH_FAILED,
+  TOKEN_EXCHANGE_FAILED,
+  USER_INFO_RETRIEVAL_FAILED,
+  ID_TOKEN_EXPIRED,
```

Comment:
This message is not very good key for translation. I would at least prefix it with SSO_